app/data/tasks/export_csv.py

```diff
@@ -55,9 +57,10 @@ def export_csv(query_key):
     # Add a directory for the schema we're outputting
     dirname = 'schema-' + str(schema.pk) + '/'
     for f, name in record_writer.get_files_and_names():
-        info = zipfile.ZipInfo(dirname + name)
+        info = zipfile.ZipInfo(filename=dirname + name, date_time=time.localtime(time.time())[:6])
```

Comment:
It'd be clearer to access the time via `struct_time`'s named parameters; I had to look it up in the documentation to figure out what this meant. https://docs.python.org/2/library/time.html#time.struct_time